ISVungleAdapter.xcframework/ios-arm64_armv7/ISVungleAdapter.framework/Headers/ISVungleAdapter.h

```diff
@@ -5,7 +5,7 @@
 #import <Foundation/Foundation.h>
 #import "IronSource/ISBaseAdapter+Internal.h"
 
-static NSString * const VungleAdapterVersion = @"5.0.0";
+static NSString * const VungleAdapterVersion = @"7.0.0.0";
```

Comment:
@ManojBudumuru-vungle @stanley-vungle  Do we need to add `-early2` here: `7.0.0.0-early2`? Thanks a lot. :)

Reply:
In my opinion, I think it should be 7.0.0 because it would follow ironSource's original syntax. I think a question needs to be asked to ironSource about their adapter version requirements. If we can change it to any string, then early2 can be added. But if they have a tool that only. uses X.X.X, then I think it should be 7.0.0.

Reply:
@stanley-vungle Thanks a lot for the detailed explanation. 👍

Reply:
@ManojBudumuru-vungle Do you know if MAX is ok with us updating the version from 3 digits to 4?